scripts/vertagen/vertagen.sh

```diff
@@ -31,14 +31,14 @@ function getVersionCandidate() {
         # v0.0.5-rc1 point to the same commit, the newer tag (v0.0.5) is used for the version number
         # generation instead of the alphabetically latest version (v0.0.5-rc1).
         local latest_tag=`git tag --sort=-taggerdate --points-at ${tagged_commit_hash} | head -n 1`
-        local version_regex='(v(.+)|([0-9]+(\.[0-9]+)*))'
+        local version_regex='((.+/)?(v([^/]+)|([0-9]+(\.[0-9]+)*)))'
```

Comment:
To capture component-specific tags (e.g. `router/v0.23.0`, `batch-ensembler/v0.0.1` etc.) It's currently not in use (as general versioning on this repo), but in the future it can help to support component-specific release cycles

Reply:
Can I request you to also leave a comment about this? It would make it easier to understand what the regex matches and why.

Reply:
Done